src/basic/util.h

```diff
@@ -61,6 +61,10 @@ static inline const char* one_zero(bool b) {
         return b ? "1" : "0";
 }
 
+static inline const char* enable_disable(bool b) {
+        return b ? "enable" : "disable";
```

Comment:
all users of this previously used the wording "enabled" and "disabled", rather than "enable" and "disable"? I think the function should be called enabled_disabled(), and return "enabled" and "disabled"...